cbits/measure_off.c

```diff
@@ -44,12 +44,16 @@
 bool has_avx512_vl_bw() {
 #if (__GNUC__ >= 7 || __GNUC__ == 6 && __GNUC_MINOR__ >= 3) || defined(__clang_major__)
   uint32_t eax = 0, ebx = 0, ecx = 0, edx = 0;
+  uint64_t xcr0;
   __get_cpuid_count(7, 0, &eax, &ebx, &ecx, &edx);
   // https://en.wikipedia.org/wiki/CPUID#EAX=7,_ECX=0:_Extended_Features
+  // __asm__("xgetbv\n\t" : "=a" (xcr0) : "c" (0));
```

Comment:
Is it supposed to be commented out?

Reply:
Oh you know what, definitely not, looks like I have to go bug someone about fixing the patch in ports also. Will force-push in a second.